src/tooltip/tooltip.js

```diff
@@ -260,6 +260,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
              * Observe the relevant attributes.
              */
             attrs.$observe( type, function ( val ) {
+              if (val && val.length >= 1 && val.charAt(0) == '@') {
+                var tag = element.find(val.length > 1 ? val.substring(1) : 'tooltip');
+                if (tag.length == 1) {
+                  ttScope.content = tag.html();
+                  tag.remove();
+                  return;
+                }
```

Comment:
There's no check for the '@' character, which is mandatory to distinguish between a tooltip content from a tooltip content reference...